src/huggingface_hub/hf_api.py

```diff
@@ -8261,6 +8277,173 @@ def grant_access(
         hf_raise_for_status(response)
         return response.json()
 
+    ###################
+    # Manage webhooks #
+    ###################
+
+    @validate_hf_hub_args
+    def get_webhook(self, webhook_id: str, token: Optional[str] = None) -> Dict:
+        """Get a webhook by its id."""
+        response = get_session().get(
+            f"{ENDPOINT}/api/settings/webhooks/{webhook_id}",
+            headers=self._build_hf_headers(token=token),
+        )
+        hf_raise_for_status(response)
+        return response.json()
+
+    @validate_hf_hub_args
+    def list_webhooks(self, token: Optional[str] = None) -> List[Dict]:
+        """List all configured webhooks."""
+        response = get_session().get(
+            f"{ENDPOINT}/api/settings/webhooks",
+            headers=self._build_hf_headers(token=token),
+        )
+        hf_raise_for_status(response)
+        return response.json()
+
+    @validate_hf_hub_args
+    def create_webhook(
+        self,
+        watched: List[WatchedItem],
+        url: str,
+        domains: List[WEBHOOK_DOMAIN_T],
+        secret: Optional[str] = None,
+        token: Optional[str] = None,
+    ) -> Dict:
+        """Create a new webhook.
+
+        Args:
+            watched (List[WatchedItem]):
+                List of items to watch. It an be users, orgs, models, datasets or spaces.
+                See `WatchedItem` for more details.
```

Comment:
```suggestion
            watched (`List[WatchedItem]`):
                List of [`WatchedItem`] to be watched by the webhook. It an be users, orgs, models, datasets or spaces.
```

2 things:
- better to have backquotes when defining the type
- setting ```[`WatchedItem`]``` in a docstring will automatically link to the object section in the docs. This is very convenient for users reading the docs on https://huggingface.co/docs/huggingface_hub/index